src/app/authentication-page/authentication-page.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="authentication-page container">
```

Comment:
get rid of or renamed `container` and `container-fluid`. as currently there is no general styles for those.